src/native-snapper.js

```diff
@@ -20,13 +18,19 @@ module.exports = class Snapper {
         `taking snapshot at path: ${outputPathWithExtension}`,
       );
 
-      const osnap = path.join(__dirname, '../node_modules/.bin/osnap');
-      const spawn = spawnSync(osnap, [`${this.platform}`, '-f', `${outputPathWithExtension}`]);
+      let proc;
+
+      if (this.platform === 'ios') {
+        proc = shell.exec(`xcrun simctl io booted screenshot ${outputPathWithExtension}`);
+      } else {
+        proc = shell.exec(`adb shell screencap -p ${outputPathWithExtension}`);
+      }
 
-      if (spawn && spawn.error) {
-        reject(spawn.error);
+      if (proc.code === 0) {
+        resolve();
       }
-      resolve();
+
+      reject(proc.stderr);
```

Comment:
Could go for something like this to increase readability over 'proc' ?

```
let nativeScreenshotCommand;

if(this.platform === 'ios'){
  nativeScreenshotCommand = `xcrun simctl io booted screenshot ${outputPathWithExtension}`
} else {
  nativeScreenshotCommand = `adb shell screencap -p ${outputPathWithExtension}`
}

const screenshotResult = shell.exec(nativeScreenshotCommand);

if(screenshotResult.code === 0){
  resolve();
} else {
  reject(screenshotResult.stderr);
}

Reply:
sounds good